browser_patches/chromium/compute_files_to_archive.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+
+import sys
+import json
+
+if len(sys.argv) < 2:
+    print "ERROR: expected arch: 32bit or 64bit"
+    sys.exit(1)
+
+if str(sys.argv[1]) == "--help" or str(sys.argv[1]) == "-h":
+    print "Usage: read_files.py [32bit|64bit] <files.cfg path>"
+    sys.exit(1)
+
+if len(sys.argv) < 3:
+    print "ERROR: expected FILE.cfg path"
+    sys.exit(1)
+
+exclude_list = [
+    # Windows exclude list
+    "chrome_child.dll",
+    "gaia1_0.dll",
+    "gcp_setup.exe",
+    "icudt.dll",
+    "interactive_ui_tests.exe",
+    "*.manifest",
+    # Linux exclude list
+    "session",
+]
+
+target_arch = sys.argv[1]
+file_name = sys.argv[2]
+
+descriptors=[]
+if sys.version_info > (3, 0):
+    exec(open(file_name).read())
+    descriptors = FILES
+else:
+    variables = {}
+    execfile(file_name, variables)
+    descriptors = variables['FILES']
+
+def filter_descriptors(entry):
+    if 'archive' in entry:
```

Comment:
shouldn't those be included?

Reply:
nah, this means that descriptor belongs to non-default archive. We care about chromium's archive only.